setup.py

```diff
@@ -156,6 +156,7 @@ def fetch_package_data():
                 "mmf_run = mmf_cli.run:run",
                 "mmf_predict = mmf_cli.predict:predict",
                 "mmf_convert_hm = mmf_cli.hm_convert:main",
+                "mmf_inference = mmf_cli.inference:inference",
```

Comment:
Tha name mmf_inference is somewhat conflicting with mmf_predict. I think the original idea was to create it in a fashion of interactive mode where the user can keep on inputting text and url and it keeps on predicting.

Reply:
Maybe `mmf-interactive`?

Reply:
Changed it to an interactive CLI and changed the name to `mmf_interactive`